### DISCUSSION TITLE
Method Illuminate\Support\Stringable::isUrl does not exist. error on the new update

### DISCUSSION THREAD

**Discussion by mannyarmghan** [livewire/livewire#7786] [7. Bugs (archived)]
### Livewire version

v3.4.0

### Laravel version

10.0

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

It was working before update.

I believe the error is here, in the "vendor\livewire\livewire\src\Mechanisms\FrontendAssets\FrontendAssets
.php : 149" File :
$url = (string) str($url)->when(! str($url)->isUrl(), fn($url) => $url->start('/'));

### Repository or Wirebox URL

nil.com

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@mannyarmghan Could you share your code which reproduces the issue with as little code as possible so we can investigate it? Thanks!

**Reply by calebporzio** [reply]
Yeah, that would be great! I see that that method indeed DOES exist, however maybe it's a new Laravel method that has been added in a minor release. Try updating your laravel/framework dependency and see if it's fixed for you.

**Reply by albertofortes** [reply]
I have exactly the same problem. Same configuration
Did you find any solution to the issue?

Maybe is not the solution you are looking for, but with a downgrade to v 3.5.5 it works:

`composer require livewire/livewire "3.3.5"`
And the problem goes.

**Reply by gurpreetkaits** [reply]
Hey,I have faced the same problem but when I did `composer update` it's gone. Any version compatibility issue with laravel?

**Reply by joshhanley** [reply]
@robinkashyap-01 yeah the `isUrl()` method was added in Laravel v10.15.0, so you need to ensure you have at least that version.

**Reply by gurpreetkaits** [reply]
yes, it worked for me. I think the issue was resolved because I have used latest version.

**Reply by joshhanley** [reply]
@robinkashyap-01 sweet! 😁

**Reply by tpetry** [reply]
I've run into the same issue.

@joshhanley Livewire's composer.json does not depend on Laravel v10.15.0. Shouldn't the version requirement be updated?

**Reply by gurpreetkaits** [reply]
> I've run into the same issue.
> 
> @joshhanley Livewire's composer.json does not depend on Laravel v10.15.0. Shouldn't the version requirement be updated?

Can you please elaborate the point?

**Reply by tpetry** [reply]
Normally, a package lists in the composer.json file the correct version a package needs to function. So Livewire should say that `laravel/framework` with at least `10.15.0` or `11.0` is required as a new method from Laravel is used. When that is done, composer would have automatically upgraded Laravel (as Livewire stated that it needs a new one). And the issue would have never happened.

**Reply by joshhanley** [reply]
@tpetry yep good idea! PRs to fix are welcome 🙂

**Comment by moh-bo** [comment]
update composer
